cli/src/main/java/org/infinispan/cli/commands/kubernetes/Uninstall.java

```diff
@@ -1,26 +1,28 @@
 package org.infinispan.cli.commands.kubernetes;
 
-import static org.infinispan.cli.commands.kubernetes.Kube.OPERATOR_SUBSCRIPTION_CRD;
-
-import java.io.IOException;
+import java.util.Map;
 
 import org.aesh.command.CommandDefinition;
 import org.aesh.command.CommandResult;
 import org.aesh.command.option.Option;
 import org.infinispan.cli.commands.CliCommand;
 import org.infinispan.cli.impl.ContextAwareCommandInvocation;
-import org.infinispan.cli.impl.KubernetesContextImpl;
+import org.infinispan.cli.impl.KubernetesContext;
+import org.infinispan.cli.logging.Messages;
+import org.infinispan.commons.util.Version;
 
+import io.fabric8.kubernetes.api.model.GenericKubernetesResource;
 import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.Resource;
 
 /**
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
  * @since 12.0
  **/
-@CommandDefinition(name = "uninstall", description = "Uninstalls the operator")
+@CommandDefinition(name = "uninstall", description = "Removes the Operator.")
```

Comment:
I think the old description was better considering the command is called "Uninstall".

Reply:
FWIW I made the change in the spirit of not using the command name to describe the command itself.

Reply:
Left it as-is